README.adoc

```diff
@@ -1,24 +1,30 @@
 == Prerequisites
 
-* a running 1.7 Kubernetes cluster with at least 4GB of memory
-these instructions assume minikube for now, for example:
+* A running 1.7 Kubernetes cluster with at least 4GB of memory
+these instructions assume minikube for now. Point your docker environment
+to the docker running in minikube.
 +
 ```
 minikube start --cpus=4 --memory=4096 --disk-size=64g
+eval $(minikube docker-env)
 ```
 
-* A working Go programming environment, with `GOPATH` set (defaults
-to `~/go`)
-
-== Build the kubernetes-model jar Enhanced with Sk8s Types
+* A working Go environment, with `GOPATH` set (defaults
+to `~/go`) and a clone of ericbottard's customized fabric8io/kubernetes-model.
```

Comment:
this is slightly confusing now (and in hindsight kind of was before), but the previous behavior was such that `GOPATH` did not need to be set *unless* something other than `~/go` was desired. Now, the script actually does require that it be set. Do we want to make the script more flexible or require it to be set? cc @ericbottard

Reply:
the script fails immediately if the GOPATH is not set so i think it's ok.
```
$ unset GOPATH
$ ./build-model 

# regenerate the fabric8 java API including k8s custom resources
pushd $GOPATH/src/github.com/fabric8io/kubernetes-model/
./build-model: line 5: pushd: /src/github.com/fabric8io/kubernetes-model/: No such file or directory
$ 
```

Reply:
`go env GOPATH` retrieves the effective value of `GOPATH`

Reply:
so should we add that to the script?

Reply:
I think we should replace occurences of `$GOPATH` with `go env GOPATH`

Reply:
👍  - i will push another commit